frontend/src/pages/offers/LineItemForm.tsx

```diff
@@ -287,7 +287,7 @@ const LineItemForm: FunctionComponent<Props> = ({
   }
 
   return (
-    <form onSubmit={submitForm} className="pb-12">
+    <form onSubmit={submitForm} className="pb-12" aria-label="Add Line Item">
```

Comment:
React Testing Library prompted me to add the aria label because otherwise it couldn't find the form. Reading up on MDN it seems like an aria label or title should be added to a form.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/form_role

Reply:
**praise:** 👍🏻